### DISCUSSION TITLE
Keychain access issue

### DISCUSSION THREAD

**Discussion by zeroes-ones** [firebase/firebase-ios-sdk#10081] [Q&A]
Hi Firebase community

We have successfully integrated Firebase into our iOS and Mac applications but on macOS we started seeing issue with Firebase keychain access and wondering if this is an issue with Firebase or did I miss any settings on macOS side. 

Any comments and help would be appreciated

<img width="684" alt="Screen Shot 2022-08-05 at 5 52 55 PM" src="https://user-images.githubusercontent.com/491437/183314656-6d7ac512-a4ee-452a-af3e-d2734684db2c.png">

**Comment by paulb777** [comment]
See https://github.com/firebase/firebase-ios-sdk/pull/8337/files. I'm not sure why that doc snippet is no longer in the latest README version. cc: @ncooke3

**Reply by zeroes-ones** [reply]
@paulb777 thanks for the info, we have already enabled keychain sharing in project, do you mean we need to add `com.firebase.FIRInstallations.installations` group in keychain sharing based on this  https://github.com/firebase/firebase-ios-sdk/pull/5603/files#diff-110dd30191f47a85c9ff632b07b7711c8b323482c3e2bad58120d1d5e2cd0110R47

**Reply by zeroes-ones** [reply]
@paulb777 we have already enabled keychain sharing, is there any specific keychain group I need to add in my project in keychain sharing?

**Reply by ncooke3** [reply]
Hi @zeroes-ones, sorry for the trouble. We have been investigating the issue and are working on a solution. We have received multiple reports of similar keychain issues. I'm using issue #9392 to give updates. 

I'll provide another update there soon with next steps! Thank you for reporting.
